### PR TITLE
Update OCS docs

### PR DESCRIPTION
**Pull request by kc1212**
The name of the paper changed and some links are broken. Also add some
text describing the differences between the paper and the
implementation.

### CODE REVIEW

**Comment by kc1212** [comment]
@ineiti please check that the added description that outlines the difference between the paper version and this implementation makes sense

File: ocs/protocol/README.md [hunk 1]
```diff
@@ -7,14 +7,14 @@ Protocols
 # Protocols
 
 The onet-framework uses protocol at its lowest level to define communication
```
Comment (jeffallen) [inline ocs/protocol/README.md]: s/protocol/protocols/

File: ocs/protocol/README.md [hunk 2]
```diff
@@ -7,14 +7,14 @@ Protocols
...
 
-- [DKG](DKG.md) - Distributed Key Generation, an implementation of the following paper:
-"Secure Distributed Key Generation for Discrete-Log
-Based Cryptosystems" by R. Gennaro, S. Jarecki, H. Krawczyk, and T. Rabin.
-- [ocs](Renecrypt.md) - onchain-secret, an implementation of the work-in-progress by
-Kokoris Kogias Eleftherios <eleftherios.kokoriskogias@epfl.ch>
+- [DKG](../../dkg/DKG.md) - Distributed Key Generation, an implementation of
+  the following paper: "Secure Distributed Key Generation for Discrete-Log
+  Based Cryptosystems" by R. Gennaro, S. Jarecki, H. Krawczyk, and T. Rabin.
+- [ocs](Renecrypt.md) - onchain-secret, an implementation of the
```
Comment (jeffallen) [inline ocs/protocol/README.md]: Why isn't this referring to the Calypso paper?

File: ocs/service/service.go [hunk 3]
```diff
@@ -1,4 +1,4 @@
-// Package service is a SCARAB implementation,  a  secure  and  decentralized
+// Package service is a CALYPSO implementation,  a  secure  and  decentralized
```
Comment (jeffallen) [inline ocs/service/service.go]: There are double-spaces where there should be single ones, I think.

**Comment by kc1212** [comment]
Fixed typos, thanks!
 
The server-side secret reconstruction is actually in the paper, in Appendix B, I've updated the README to reflect that.

**Pull request merged**